cloud_functions/README.md

```diff
@@ -0,0 +1,15 @@
+# Bard - Cloud Functions
+Cloud functions related to metrics
+
+## Overview
+This is a collection of cloud functions that augment the core bard service. The main service is a set of endpoints intended to be called from clients, either anonymously or with an access token that identifies the user performing the action. The cloud functions here may have other triggers, such as a Google Pub/Sub message.
+
+## Function Documentation
+
+### `flagCryptominer`
+`flagCryptominer` watches for messages on the `terra-bard-[env]/topics/cryptominers` Pub/Sub topic containing IDs of identified cryptominer accounts and sets the "Is Cryptominer" flag in their Mixpanel profile to `true`.
```

Comment:
topic will be `"projects/{{$googleProj}}/topics/terra-cryptomining"` where `googleProj` is `broad-dsde-<env>`